fit/core-reference/src/test/java/org/apache/syncope/fit/AbstractITCase.java

```diff
@@ -820,6 +821,31 @@ protected void removeLdapRemoteObject(
         return object;
     }
 
+    protected CASSPTO buildCASSP() {
```

Comment:
Maybe this method can be moved to `ClientAppITCase` as no one else is using it ATM